ui/src/react-admin/modules/content-page/components/blocks/BlockContentEnclose/hooks/useGetContentBlockEncloseContent.types.ts

```diff
@@ -0,0 +1,39 @@
+import { IconNameSchema } from '@viaa/avo2-components/dist/components/Icon/Icon.types';
+
+export interface GetContentBlockEncloseContentReturnType {
+	id?: string;
+	name?: string;
+	description?: string;
+	thumbnail?: string;
+	dateCreated?: string;
+	maintainerName?: string;
+	objectType?: string;
+	identifier?: string;
+	link?: string;
+	pid?: string;
+	type?: 'IE_OBJECT' | 'CONTENT_PAGE';
+}
+
+export interface IeObject {
+	maintainerId: string;
+	name: string;
+	title: string;
+	description: string;
+	thumbnailUrl: string;
+	dateCreatedLowerBound: string;
+	maintainerName: string;
+	ebucoreObjectType: string;
+	meemooIdentifier: string;
+	maintainerSlug: string;
+	schemaIdentifier: string;
+	dctermsFormat: string;
+	objectType: string;
+}
+
+export interface ContentPage {
+	id: string;
+	title: string;
+	description: string;
+	thumbnailPath: string;
+	path: string;
+}
```

Comment:
Instead of defining this interface which is a partial duplicate of ContentPageInfo, you can pick the fields that you need from the full ContentPageInfo interface that already exists.

The advantance of this is that if one of the properties changes, we also get typescript changes for this type here.

```suggestion
export type ContentPage = Pick<ContentPageInfo, 'id' | 'title' | 'description' | 'thumbnailPath' | 'path'>
```